README.md

```diff
@@ -68,6 +68,9 @@ you will see what steps the library is doing and understand why a session you
 expect to be there is not present. For extra details, you can also enable `trace`
 level logging.
 
+Note: instead of using `get` and `set` as seen above, you may also wish to get and set properties on the
+session object natively in order to make your code compatible with other libraries ie `request.session.data = request.body` and `const data = request.session.data` are also possible.
```

Comment:
this should be slightly reworded. get and set are methods, so please refer to them as such. the new way you introduced can be described as "using property getters and setters"